tests/ImageSharp.Tests/Common/SimdUtilsTests.cs

```diff
@@ -133,7 +133,7 @@ static void RunTest(string serialized) => TestImpl_BulkConvertByteToNormalizedFl
         FeatureTestRunner.RunWithHwIntrinsicsFeature(
             RunTest,
             count,
-            HwIntrinsics.AllowAll | HwIntrinsics.DisableAVX512F | HwIntrinsics.DisableAVX2 | HwIntrinsics.DisableSSE41);
```

Comment:
This change is causing problems

Reply:
@JimBobSquarePants It would be cool If you could look over this one. When I change DisableSSE42 to DisableHWIntrinsics I will get test failures

Reply:
In this case we want to drop `HwIntrinsics.DisableSSE41` entirely. This is because we are explicitly testing the `HwIntrinsics` version of the method and disabling intrinsics disables the entire `Vector128` path. 

Our previous test flags ensured we were covering both `Sse41` and `Sse2` on Windows but I think our cross-platform test will ensure coverage in this case since we're using `Vector128<T>` directly.